src/AlgAss/AlgGrp.jl

```diff
@@ -31,6 +31,20 @@ group(A::GroupAlgebra) = A.group
 
 has_one(A::GroupAlgebra) = true
 
+##To be removed
+struct InfiniteDimensionError <: Exception
+end
+
+##To be removed
+function Base.showerror(io::IO, err::InfiniteDimensionError)
+  println(io, "Infinite-dimensional vector space")
+end
+
+function vector_space_dim(A::GroupAlgebra{T, S, R}) where {T <: FieldElem, S, R}
+  isfinite(group(A)) || throw(InfiniteDimensionError())
+  return order(group(A))
```

Comment:
```suggestion
  return dim(A)
```

Reply:
I think the idea behind all of this `dim` business is to do it the other way around. Aka with this change, `dim` should delegate to `vector_space_dim`. (But IMO this should be in this PR @emikelsons)

Another thing: `order(group(A))` will return a ZZRingElem. I haven't followed what other `vector_space_dim` methods return, if Int or ZZRingElem. In any case, this should be consistent.

For Ints, you would do
```suggestion
  return order(Int, group(A))
```